app.js

```diff
@@ -167,14 +167,11 @@ app.get('/healthcheck', function(req, res) {
   });
 });
 
+if (!env.get('WEBMAKER_REDIRECT_URL')) {
```

Comment:
@Pomax ah, is audience always pointing at a webmaker.org instance? (e.g. localhost:whatever)